logrusx/helper.go

```diff
@@ -107,6 +107,25 @@ func (l *Logger) WithRequest(r *http.Request) *Logger {
 	return ll
 }
 
+func (l *Logger) WithSpan(sc trace.Span) *Logger {
```

Comment:
Did you consider passing the full context here instead and then using `trace.SpanFromContext`? I'd guess that that would be the most universal usage.

Reply:
Done.